honeybee_radiance_postprocess/leed.py

```diff
@@ -0,0 +1,228 @@
+            each sensor grid. If None, the files will not be written and only
+            the summary dictionary will be calculated.
+
+    Returns:
+        A dictionary with a summary of LEED credits in the format below. All
+        percentages are between 0 and 100 and the floor areas are in the units system
+        of the HBJSON model. If no sensor grid meshes were found in the HBJSON
+        model and no areas could be associated with each mesh face, the output
+        will not contain floor_area keys and each sensor will be assumed to
+        occupy a similar area.
+
+    .. code-block:: python
+
+        {
+            "credits": 2,
+            "percentage_passing": 76.2,
+            "percentage_passing_9AM": 78.5,
+            "percentage_passing_3PM": 82.4,
+            "sensor_count_passing": 762,
+            "sensor_count_passing_9AM": 785,
+            "sensor_count_passing_3PM": 824,
+            "total_sensor_count": 1000,
+            "floor_area_passing": 762.0,
+            "floor_area_passing_9AM": 785.0,
+            "floor_area_passing_3PM": 824.0,
+            "total_floor_area": 1000.0
+        }
+    """
+    # first load the results into pass/fail matrices of ones/zeros
+    res_folder_9 = os.path.join(folder, '9AM', 'results')
+    res_folder_3 = os.path.join(folder, '3PM', 'results')
+    pass_fail_9 = ill_pass_fail_from_folder(res_folder_9, glare_control, grids_filter)
+    pass_fail_3 = ill_pass_fail_from_folder(res_folder_3, glare_control, grids_filter)
+
+    # determine which sensors pass for both hours
+    pass_fail_comb = []
+    for p_fails9, p_fails3 in zip(pass_fail_9, pass_fail_3):
+        p_fails_comb = []
+        for pf9, pf3 in zip(p_fails9, p_fails3):
+            if pf9 == 1 and pf3 == 1:
+                p_fails_comb.append(1)
+            else:
+                p_fails_comb.append(0)
+        pass_fail_comb.append(p_fails_comb)
+
+    # write the pass/fail criteria into the sub-directory if specified
+    if sub_folder:
+        _pass_fail_to_files(
+            folder, sub_folder, pass_fail_comb, pass_fail_9, pass_fail_3, grids_filter)
+
+    # setup the summary dictionary with the results
+    summary_dict = {
+        'sensor_count_passing': sum(sum(pf) for pf in pass_fail_comb),
+        'sensor_count_passing_9AM': sum(sum(pf9) for pf9 in pass_fail_9),
+        'sensor_count_passing_3PM': sum(sum(pf3) for pf3 in pass_fail_3),
+        'total_sensor_count': sum(len(pf9) for pf9 in pass_fail_9)
+    }
+
+    # next, check to see if there is a HBJSON with sensor grid meshes for areas
+    grid_areas = []
+    for base_file in os.listdir(folder):
+        if base_file.endswith('.hbjson') or base_file.endswith('.hbpkl'):
+            hb_model = Model.from_file(os.path.join(folder, base_file))
+            filt_grids = _filter_by_pattern(
+                hb_model.properties.radiance.sensor_grids, filter=grids_filter)
```

Comment:
@chriswmackey, this will create a circular dependency - if we want to access the sensor grids in a model then we need to have honeybee-radiance as a dependency which will be against what you had in mind for making honeybee-radiance-postporcess a dependency of the honeybee-radiance.

I didn't notice this until it caused the tests to fail. I'm not sure what is the best approach here to resolve this. The simplest approach is to make honeybee-radiance a dependency of honeybee-radiance-postprocess. The only undesired change is that `honeybee-radiance postprocess` commands in the cli will change to `honeybee-radiance-postprocess`. Let me know what you think.

Reply:
We decided to change the dependency here and make `honeybee-radiance` a dependency of `honeybee-radiance-postprocess`